vault/identity_store_util.go

```diff
@@ -147,6 +170,74 @@ func (c *Core) loadIdentityStoreArtifacts(ctx context.Context) error {
 	return loadFunc(ctx)
 }
 
+func (i *IdentityStore) encodeEntity(entity *identity.Entity) (*storagepacker.Item, error) {
+	if entity == nil {
+		return nil, fmt.Errorf("nil entity")
+	}
+
+	entityJSON, err := jsonutil.EncodeJSON(entity)
```

Comment:
I think we can and still want to encode this as a protobuf. Just using proto.Marshal instead of proto.MarshalAny.

Reply:
Fixed.